skymp5-client/src/services/services/networkingService.ts

```diff
@@ -79,118 +79,113 @@ export class NetworkingService extends ClientListener {
+            this.controller.emitter.emit("anyMessage", event);
+          }
+          else if (msgAny.t === MsgType.Teleport) {
+            const event = { message: msgAny };
+            this.controller.emitter.emit("teleportMessage", event);
+            this.controller.emitter.emit("anyMessage", event);
+          }
+          else if (msgAny.t === MsgType.UpdateProperty) {
+            const event = { message: msgAny };
+            this.controller.emitter.emit("updatePropertyMessage", event);
+            this.controller.emitter.emit("anyMessage", event);
+          }
+          else if (msgAny.t === MsgType.DeathStateContainer) {
+            const event = { message: msgAny };
+            this.controller.emitter.emit("deathStateContainerMessage", event);
+            this.controller.emitter.emit("anyMessage", event);
+          }
+          else if (msgAny.t === MsgType.CreateActor) {
+            const event = { message: msgAny };
+            this.controller.emitter.emit("createActorMessage", event);
+            this.controller.emitter.emit("anyMessage", event);
+          }
+          else if (msgAny.t === MsgType.CustomPacket) {
+            const event = { message: msgAny };
+            this.controller.emitter.emit("customPacketMessage2", event);
+            this.controller.emitter.emit("anyMessage", event);
+          }
+          else if (msgAny.t === MsgType.DestroyActor) {
+            const event = { message: msgAny };
+            this.controller.emitter.emit("destroyActorMessage", event);
+            this.controller.emitter.emit("anyMessage", event);
+          }
+          else if (msgAny.t === MsgType.HostStart) {
+            const event = { message: msgAny };
+            this.controller.emitter.emit("hostStartMessage", event);
+            this.controller.emitter.emit("anyMessage", event);
+          }
+          else if (msgAny.t === MsgType.HostStop) {
+            const event = { message: msgAny };
+            this.controller.emitter.emit("hostStopMessage", event);
+            this.controller.emitter.emit("anyMessage", event);
+          }
+          else if (msgAny.t === MsgType.SetInventory) {
+            const event = { message: msgAny };
+            this.controller.emitter.emit("setInventoryMessage", event);
+            this.controller.emitter.emit("anyMessage", event);
+          }
+          else if (msgAny.t === MsgType.SetRaceMenuOpen) {
+            const event = { message: msgAny };
+            this.controller.emitter.emit("setRaceMenuOpenMessage", event);
+            this.controller.emitter.emit("anyMessage", event);
+          }
+          else if (msgAny.t === MsgType.SpSnippet) {
+            const event = { message: msgAny };
+            this.controller.emitter.emit("spSnippetMessage", event);
+            this.controller.emitter.emit("anyMessage", event);
+          }
+          else if (msgAny.t === MsgType.UpdateGamemodeData) {
```

Comment:
![](https://memepedia.ru/wp-content/uploads/2018/11/mozhno-nenado-mem-9.jpg)

Reply:
no shitpost in gh please, more context